zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/repo/ExporterDescriptor.java

```diff
@@ -39,4 +39,8 @@ public ExporterConfiguration getConfiguration() {
   public String getId() {
     return configuration.id();
   }
+
+  public boolean isSameTypeAs(final ExporterDescriptor other) {
+    return exporterClass.equals(other.exporterClass);
+  }
```

Comment:
This was added so that we can use it to verify if the exporter can be initialized from the given other exporter. But I realized that it makes sense to verify this in the PartitionManager when applying the enable operation. It can fail the operation even before delegating it to the ExporterDirector. So this method is currently unused.